hw2/css/style.css

```diff
@@ -0,0 +1,169 @@
+.clearfix:after {
+	content: '';
+	clear: both;
+	display: table;
+	overflow: hidden;
+}
+
+a{
+	text-decoration: none;
+}
+
+img{
+	width: 100%;
+}
+
+body{
+	background-color: #f9f9f9;
+}
+
+	.container{
+		background-color: #FFF;
+		margin: 0 auto;
+		width: 78%;
+		box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+		padding: 50px 48px 60px 46px;
+	}
+
+		.header__logo{
+			width: 145px;
+			height: 40px;
+			margin-bottom: 40px;
+		}
+
+		.menu-top{
+			display: flex;
+			line-height: 40px;
+			width: 100%;
+			background-color: #fff;
+		    background-image: -webkit-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: -o-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+		}
+
+			.menu-top__item{
+				text-shadow: 5px 5px 7px rgba(0, 0, 0, 0.8);
+				font-family: "Open Sans", "Fira Sans", Arial, sans-serif;
+				padding: 0 4%;
+				color: #fff;
```

Comment:
I don´t know why, but it just doesn't want to do that. I tried to change, and all child elements became blue...